ncm-metaconfig/src/main/metaconfig/haproxy/pan/schema.pan

```diff
@@ -198,13 +198,20 @@ type haproxy_service_bind = {
     'port' ? type_port
 };
 
+type haproxy_service_frontend_errorfile = {
+    'code' : long(200..600)
+    'filename' : absolute_file_path
+};
+
 type haproxy_service_frontend = {
     'acl' ? dict()
     'bind' : haproxy_service_bind[]
     'default_backend' : string
+    'use_backend' ? string[]
```

Comment:
```suggestion
    'use_backend' ? string_trimmed[]
```
This also needs additional validation on `haproxy_service` to ensure that the backend has been defined, similar to the logic for `default_backend`.